modin/numpy/utils.py

```diff
@@ -19,12 +19,7 @@
 _INTEROPERABLE_TYPES = (pd.DataFrame, pd.Series)
 
 
-def try_convert_from_interoperable_type(obj):
+def try_convert_from_interoperable_type(obj, copy=True):
```

Comment:
Given that the default behavior of converting to NumPy from pandas is copy=False, should we make copy False here? This is used in the conversion for when we call NumPy methods on Modin objects that aren't arrays.

Reply:
I would say yes, as defaulting to `True` here would break the scenario of modifying pandas objects from which numpy arrays were made...

Reply:
Sounds good to me, although we set `copy=False` explicitly anywhere where the array (and hence the df) should be modified, and in the other places, the array doesn't get modified (so neither should the df).

Reply:
with that said, maybe we should not provide _any_ default value, hence requiring the user of this function to decide if they need a copy or not every time?